compiler/optimizer/OrderBlocks.cpp

```diff
@@ -1254,7 +1254,7 @@ bool TR_OrderBlocks::peepHoleBranchToLoopHeader(TR::CFG *cfg, TR::Block *block,
    }
 
 // apply peephole optimizations to blocks that end in a branch and branch around a single goto
-void TR_OrderBlocks::peepHoleBranchAroundSingleGoto(TR::CFG *cfg, TR::Block *block, char *title)
+void TR_OrderBlocks::peepHoleBranchAroundSingleGoto(TR::CFG *cfg, TR::Block *block, const char *title)
    {
    TR_ASSERT(block->endsInBranch(), "peepHoleBranchBlock called on block that doesn't end in a branch!");
```

Comment:
I just noticed a typo. in this `TR_ASSERT` - it mentions `peepHoleBranchBlock` rather than `peepHoleBranchAroundSingleGoto`.  May I ask you to correct it while you're here?